Data_Structures_Answers.md

```diff
@@ -2,17 +2,32 @@ Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
 
+O(1)
+
 2. What is the space complexity of your ring buffer's `append` function?
 
+O(1)
+
 3. What is the runtime complexity of your ring buffer's `get` method?
 
+O(n)
+
 4. What is the space complexity of your ring buffer's `get` method?
 
+O(n)
 
 5. What is the runtime complexity of the provided code in `names.py`?
 
+O(n^2)
+
 6. What is the space complexity of the provided code in `names.py`?
 
+O(n)
+
```

Comment:
Nailed the questions Kyle 👍  The space complexity will end up being `0(1)` since no extra space is allocated.